services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Utils.kt

```diff
@@ -110,3 +112,9 @@ fun String?.isValidBusinessId(): Boolean {
         false
     }
 }
+
+fun MultipartFile.validNameAndType() =
+    Pair(
+        AttachmentValidator.validFilename(originalFilename),
+        AttachmentValidator.ensureMediaType(contentType)
+    )
```

Comment:
Shouldn't this be in AttachmentValidator?

Reply:
Moved.